.github/workflows/tests.yml

```diff
@@ -11,12 +11,12 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [ "8.0", "8.1", "8.2", "8.3" ]
+                php: [ "8.0", "8.1", "8.2", "8.3", "8.4" ]
```

Comment:
The services repo does not depend on `Auth` but this one does, shouldn't we remove the "8.0" support?

Reply:
Oh sorry, I miss-interpreted the conversation in our chat, I thought we were going to remove support here instead of my own PR.

Approved.